dask_sql/physical/utils/filter.py

```diff
@@ -91,7 +91,7 @@ def attempt_predicate_pushdown(ddf: dd.DataFrame) -> dd.DataFrame:
     try:
         return dsk.layers[name]._regenerate_collection(
             dsk,
-            new_kwargs={io_layer: {"filters": filters}},
+            new_kwargs={io_layer: {"filters": filters, "index": False}},
```

Comment:
Looks like the issues with predicate pushdown were stemming from the automatic setting of an index in `read_parquet` by default, which this kwarg override should disallow.

Chatting with @rjzamora, we agreed that this shouldn't be the default behavior, so we may be able to remove this override later on when changes are made upstream.